examples/stable/test/end_to_end/candid_rpc/motoko_examples/cert-var/test/tests.ts

```diff
@@ -1,18 +1,18 @@
 import { ActorSubclass, Certificate, HttpAgent } from '@dfinity/agent';
 import { IDL, lebDecode, PipeArrayBuffer } from '@dfinity/candid';
 import { Principal } from '@dfinity/principal';
-import {
-    createAuthenticatedAgent,
-    getCanisterId,
-    whoami
-} from 'azle/_internal/dfx';
+import { getCanisterId } from 'azle/_internal/dfx';
 import {
     defaultPropTestParams,
     expect,
     getCanisterActor,
     it,
     Test
 } from 'azle/_internal/test';
+import {
+    createAuthenticatedAgent,
+    whoami
+} from 'azle/experimental/_internal/dfx';
```

Comment:
Make sure that we have the experimental check at the top of `_internal/dfx`

Reply:
I am wondering if this is an indication that dfx should be in lib not build?

Reply:
done